packages/ckeditor5-font/tests/ui/colorui.js

```diff
@@ -186,6 +186,20 @@ describe( 'ColorUI', () => {
 				sinon.assert.calledWithExactly( spy, 'testColorCommand', sinon.match( { value: '#a37474' } ) );
 			} );
 
+			it( 'should cancel changes', () => {
+				dropdown.isOpen = false;
+
+				dropdown.colorTableView.selectedColor = '#ff0000';
+
+				dropdown.isOpen = true;
+
+				dropdown.colorTableView.selectedColor = '#123456';
+
+				dropdown.colorTableView.cancelButtonView.element.dispatchEvent( new Event( 'click' ) );
```

Comment:
To test editor `API` we could change dispatching this event into fire the `cancel` button like this: 
```js
dropdown.colorTableView.cancelButtonView.fire( 'execute' );
```